cmd/end2endtest/helpers.go

```diff
@@ -393,6 +448,76 @@ func (t *e2eElection) sendVote(v voteInfo, apiClientMtx *sync.Mutex) (int, error
 	return contextDeadline, nil
 }
 
+// ballotVotes from a default list of 10 vote values that exceed the max value, max total cost and is not unique will
+func ballotVotes(b ballotData, nvotes int) ([][]int, [][]*types.BigInt) {
+	votes := make([][]int, 0, nvotes)
+	var resultsCh1, resultsCh2 []*types.BigInt
```

Comment:
it's not immediately clear to me what "choice" ("Ch") means in this var name: are you talking about "Field" or "Value"? (in the terms described in https://developer.vocdoni.io/protocol/ballot)